### PR TITLE
docs: Update changelog

### PR DESCRIPTION
**Pull request by TxHawks**
## Description
Update changelog for 10.1.2 release

## Versioning

- [ ] Major
- [ ] Minor
- [ ] Patch

## Checklist

- [x] The code was formatted using Prettier (`yarn run format`)
- [x] The code has no linting errors (`yarn run lint`)
- [x] All tests are passing (`yarn run test`) 
- [x] There are no flow-type errors (`yarn run flow`)

#### Changes
If one of the following checks doesn't make sense due to the type of PR, just check them.

- [x] Tests have been added/updated
- [x] Documentation has been added/updated
- [x] My changes have proper flow-types

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ If you're searching for older logs, please check the [old changelog](https://git
 
 ## 10.0
 
+### 10.1.1
```
Comment (rofrischmann) [inline CHANGELOG.md]: 10.1.2 :P

**Comment by rofrischmann** [comment]
In general you can push changelog stuff to master if you want to

File: CHANGELOG.md (hunk 1 above)
Comment (TxHawks) [reply CHANGELOG.md]: Oops

**Comment by TxHawks** [comment]
Fixed. 

Master is a protected branch, I don't have permissions to push to it.

**Comment by rofrischmann** [comment]
Oh really. I might open that for you...

**Pull request merged**